ExternalModule.php

```diff
@@ -41,10 +44,7 @@ function isRepeatEntireEvent() {
      * Returns true if the event is a "Repeat Instrument (repeat independently of each other)". 
      */
     function isRepeatInstrument() {
-        if ($_GET['instance'] > 1 && !isset($_GET['oldinstance'])) {
-            return true;
-        } 
-        return false;
+        return $GLOBALS['isRepeatingForm'];
```

Comment:
@mbentz-uf  This causes APF to fail to populate fields for an event that is _after_ a repeating form.

![image](https://user-images.githubusercontent.com/20332546/128374640-2dae0634-f152-4cd9-acf1-d6c860d5a143.png)

If I revert this change, event 4 populates from event 3. However, if I add a 2nd instance to event 3, event 4 no longer autopopulates.